docs/topics/exporters.rst

```diff
@@ -236,7 +236,7 @@ PythonItemExporter
 XmlItemExporter
 ---------------
 
-.. class:: XmlItemExporter(file, item_element='item', root_element='items', \**kwargs)
+.. class:: XmlItemExporter(file, *, item_element='item', root_element='items', list_item_element='value', **kwargs)
```

Comment:
The new `*` is a documentation fix, because the class uses `**kwargs` for the following keyword arguments, which means they cannot be specified positionally.

I’m not sure why `\**` was there, but it is showing as such in the rendered documentation, and removing it works.

These both as issues that affect the code base as a whole, so I’ll create separate issues to deal with them globally.